app/http.php

```diff
@@ -1272,7 +1272,17 @@ function (string $runtimeId, ?string $payload, string $path, string $method, mix
                     throw new Exception($executionResponse['error'], 400);
                 }
 
-                // Unknown error
+                // Cleanup frequent errors
+                if ($executionResponse['errNo'] === CURLE_COULDNT_RESOLVE_HOST) {
+                    $executionResponse['error'] = 'Could not resolve host';
+                    $log->addTag('hostname', $hostname);
+                    $log->addTag('port', '3000');
+                } elseif ($executionResponse['errNo'] === CURLE_COULDNT_CONNECT) {
+                    $executionResponse['error'] = 'Could not connect to host';
+                    $log->addTag('hostname', $hostname);
+                    $log->addTag('port', '3000');
+                }
+
                 throw new Exception('Internal curl errors has occurred within the executor! Error Number: ' . $executionResponse['errNo'] . '. Error Msg: ' . $executionResponse['error'], 500);
             }
```

Comment:
```suggestion
               $log->addExtra('error', $executionResponse['error']);
               $log->addTag('hostname', $hostname);
             
                throw new Exception('Internal curl errors has occurred within the executor! Error Number: ' . $executionResponse['errNo'], 500);
            }
```

Can we do something like this? 

And if its just the `error` and `hostname` we're adding, can we just not add it in the central place where we add the other logs?